Sources/Tests/CoreTests/test_plCmdParser.cpp

```diff
@@ -547,3 +547,17 @@ TEST(plCmdParser, case_sensitive_match)
 
     EXPECT_EQ(specified, true);
 }
+
+TEST(plCmdParser, seperator_in_value)
```

Comment:
```suggestion
TEST(plCmdParser, separator_in_value)
```
Extremely pedantic spelling correction, but this one's a bit of a pet peeve. Sorry. ;)

Reply:
Fixed. (Should have just hit the commit suggestion button but noticed it too late!)